common-libs/user-management/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/user-management",
-  "version": "1.5.0",
+  "version": "2.0.0",
```

Comment:
despite this library uses libraries that have to be update with major version, this library (user-management) doesn't use bloom-vault under the hood, so my opinion **to update it with a minor update**

Reply:
changed